ui/components/MesheryEventViewer.js

```diff
@@ -11,37 +11,36 @@ const styles = (theme) => ({
   },
 });
 
-class MesheryEventViewer extends React.Component {
+function MesheryEventViewer(props) {
   state = {
     dialogShow: false,
   }
 
   handleSnackbarClose = (_, reason) => {
     if (reason === 'clickaway') return;
 
-    this.props.deleteEvent();
+    props.deleteEvent();
   };
 
-  render() {
-    const {
-      classes, eventVariant, eventSummary, eventDetails,
-    } = this.props;
-
-    return (
-      <NoSsr>
-        <React.Fragment>
-          <MesherySnackbarWrapper
-            key={`event_-_${eventVariant}`}
-            variant={eventTypes[eventVariant] ? eventTypes[eventVariant].type : eventTypes[0].type}
-            message={eventSummary}
-            details={eventDetails}
-            onClose={this.handleSnackbarClose}
-            className={classes.event}
-          />
-        </React.Fragment>
-      </NoSsr>
-    );
-  }
+
+  const {
+    classes, eventVariant, eventSummary, eventDetails,
+  } = props;
+
+  return (
+    <NoSsr>
+      <>
```

Comment:
@afzal442 Is this empty tag required?

Reply:
yes, this is a kind of syntactic sugar. have a look here https://reactjs.org/docs/fragments.html#short-syntax. thanks.